handbook/company/pricing-features-table.yml

```diff
@@ -447,7 +448,9 @@
       tier: Free
       usualDepartment: IT
       waysToUse:
-        - description: Deploy custom configuration profiles on macOS and Windows (coming soon) and verify that they're installed.
+        - description: Deploy custom profiles on macOS and verify that they're installed. Windows coming soon (2023-12-31). #13281
```

Comment:
Mike: "configuration profiles" is clearer

Reply:
✅